### DISCUSSION TITLE
[Feature] <upload birthdays from contacts>

### DISCUSSION THREAD

**Discussion by shomilsaini** [immich-app/immich#10372] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Option to add birthdays of all identified people from phone contacts or contacts import via vCard/CSV match via name or give option to choose.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by shomilsaini** [comment]
I saw there is an option to add birthdays one by one via [people](https://demo.immich.app/people) page by clicking on the face 3-dots, I would like to see bulk birthday imports via contacts.

Thanks

**Comment by bo0tzz** [comment]
#2629
